### ISSUE TITLE
Apps system broken: imgur, giphy apps not returning results

### ISSUE THREAD

**Issue by bkraul** [RocketChat/Rocket.Chat#13957]
### Description:
The apps system seems to be broken. The apps listing shows `$NaN` on status of not-yet installed apps. Attempting to install an app returns a 404 error. Also if imgur and giphy apps are already installed, using them generates a `no results found` message.

### Steps to reproduce:
1. Update to latest develop docker image (849c9d1)
2. Go to Administration - Apps. 
3. See the broken list.
4. Attempt to install an app. Will get 404 error.
5. If imgur or giphy apps are installed, try a search. "no results found" will be returned. (Try `cat` ... there *should* be results.

### Expected behavior:
Apps should show correct status. Apps should be able to be installed. Giphy and Imgur apps should return results.
### Actual behavior:
Described in steps above.

**UPDATE:** Last worked OK on docker **develop** build based on commit 7c2d680.

### Screenshots
App List:
![image](https://user-images.githubusercontent.com/14616851/55282374-ca4ae980-5310-11e9-9033-f8e9a9e416e1.png)
Installing an app:
![image](https://user-images.githubusercontent.com/14616851/55282380-d5057e80-5310-11e9-9ef7-1b989a15fe2a.png)
No results on giphy app:
![image](https://user-images.githubusercontent.com/14616851/55282382-dd5db980-5310-11e9-9236-e0d68f379837.png)

### Rocket.Chat

|                 Version | 1.0.0-develop                                             |
| ----------------------: | --------------------------------------------------------- |
|     Apps Engine Version | 1.4.0                                                     |
|      Database Migration | 139                                                       |
| Database Migration Date | Sat Mar 30 2019 17:08:43 GMT-0500 (Central Daylight Time) |
|            Installed at | Tue Jul 03 2018 10:28:32 GMT-0500 (Central Daylight Time) |
|                  Uptime | 34 seconds                                                |
|           Deployment ID | wecGmPkstnsm2e7ge                                         |
|                     PID | 20                                                        |
|       Running Instances | 1                                                         |
|                   OpLog | Enabled                                                   |

### Commit

|    Hash | c13708a3e4eaed3ed1aaf74995497a6d56af861b                |
| ------: | ------------------------------------------------------- |
|    Date | Thu Mar 28 20:17:28 2019 +0530                          |
|  Branch | develop                                                 |
|     Tag | 0.74.2                                                  |
|  Author | Kautilya Tripathi                                       |
| Subject | [FIX] Read Receipt for Livechat Messages fixed (#13832) |

### Runtime Environment

|         OS Type | Linux                        |
| --------------: | ---------------------------- |
|     OS Platform | linux                        |
|         OS Arch | x64                          |
|      OS Release | 4.15.0-46-generic            |
|    Node Version | v8.11.4                      |
|       OS Uptime | 16 days, 5 hours, 13 seconds |
| OS Load Average | 1.03, 0.57, 0.76             |
| OS Total Memory | 7.79 GB                      |
|  OS Free Memory | 133.62 MB                    |
|    OS CPU Count | 4                            |

### Build Environment

|  OS Platform | linux                  |
| -----------: | ---------------------- |
|      OS Arch | x64                    |
|   OS Release | 4.15.0-1027-aws        |
| Node Version | v8.11.4                |
|         Date | March 28, 2019 9:52 AM |

**Comment by graywolf336** [comment]
This has been resolved

**Comment by bkraul** [comment]
What commit or PR was this resolved on?

**Comment by bkraul** [comment]
Fixed in develop docker build based on commit 408aa9d .

**Comment by graywolf336** [comment]
Because it was not related to anything inside of Rocket.Chat and is a private repository. Thus if I linked you to the change you wouldn't be able to say it. So yes, I am going to keep it closed and resolve issues like this when it is outside of the scope of Rocket.Chat.

**Comment by geekgonecrazy** [comment]
To be clear.. this pulls from: marketplace.rocket.chat - the source code for this is not currently open source.  

So we made the fix to our publicly running service and actually deployed the commit that shows the price of the app. (which in this case all apps are free)

Marketplace.rocket.chat will in the near future allow developers to list apps to sell so this is what these changes are bringing about.

The changes landed on develop a tad bit ahead of our changes to the marketplace being deployed.  But the changes have now been deployed so this particular issue is now resolved.
